doc/examples/applications/plot_human_mitosis.py

```diff
@@ -22,13 +22,11 @@
 from scipy import ndimage as ndi
 
 from skimage import (
-    color, feature, filters, io, measure, morphology, segmentation, util
+    color, feature, filters, measure, morphology, segmentation, util
 )
-from skimage.data import image_fetcher
+from skimage.data import human_mitosis
 
-
-path = image_fetcher.fetch('data/mitosis.tif')
-image = io.imread(path)
+image = human_mitosis()
```

Comment:
We've been discussing "just" mitosis for this but actually, in biological circles, "mitosis" refers to a well known dataset that ships with Fiji, pictured in my napari announcement video:

https://twitter.com/jnuneziglesias/status/1187400066987741186

(a) we should probably add that dataset to skimage, but
(b) let's not use just mitosis. I'd rather "mitosis2d", for example?

Reply:
@emmanuelle 👆

Reply:
> (b) let's not use just mitosis. I'd rather "mitosis2d", for example?


:-1:  we said we add "3d" to confocal images earlier.

Reply:
> :-1: we said we add "3d" to confocal images earlier.

:thinking: we're appending "3d" because the image is 3D, not necessarily because it's from confocal fluorescence microscopy (unless I missed something). So appending "2d" would be consistent...

Reply:
yes, but 2D images do not have 2d tags.

Reply:
I said confocal because so far, we have only confocal images and confocal microscopes provide 3D images.

Reply:
To clarify, having cell / cell3d and mitosis2d / mistosis, It sounds bad in terms of consistency and given the effort we are putting to fix former choices like this in the library, I prefer to put a veto now :)

Reply:
> having cell / cell3d

Wait, we have `cell` and (with this PR) `cells3d` (formerly `cells` but unregistered so far) :)

> and mitosis2d / mitosis

I see what you mean now.

Reply:
@jni but if we integrate the Fiji dataset at some point, can't we call it `mitosis3d` ? Since we document our datasets with specific gallery examples, I don't think it's a big problem that we call mitosis something else than what Fiji does.

Reply:
Well, it would be mitosis5d. ;) imho if we can aim for consistency with our neighbours, we should. As pointed out by @mkcor, we don't yet have `cells3d`, so we can try to think of something better that maintains our own consistency.

@sciunto 

> I prefer to put a veto now :)

Veto is such a strong word — maybe prefer to have a robust discussion to arrive at the optimal choice now? =) I agree with you that `cell`/`cells3d` vs `mitosis2d`/`mitosis` sounds bad. Perhaps we can come up with something better within the constraints below. If we can't, *then* we can start to debate whether we should remove some constraints:

- the use of the `nd` suffix should be consistent.
- we should avoid using `fetch` in examples.
- `mitosis` should be used for the "famous" mitosis dataset.

I think possibly the last constraint, "datasets should only have one word", could be relaxed, and then `human_mitosis` can be `mitosis_screen` (it is an example from a high-content imaging screen). 

I'd propose an additional constraint that would make our lives harder, but maybe it can work:

- names should not be prefixes of other names

since this makes autocomplete annoying. For `cells3d`, we could use `fibroblasts` for example (I think that's what they were, otherwise we can adjust 😅).